docs/protocol/cli.md

```diff
@@ -141,6 +141,11 @@ Some older processors/VMs are no longer supported by official releases, but can
 
 2. Open Powershell, type `cd Downloads` (or `cd Your-File-Location`).
 
+:::warning
+Windows may block Subspace because of unidentified developer restrictions. To resolve this, select 'Allow access'
```

Comment:
The warning is because the application is trying to access the internet. This is expected as it's how the farmer talks to other farmers on the network.